src/main/java/org/betonquest/betonquest/api/profile/UUIDProfileProvider.java

```diff
@@ -12,24 +10,20 @@
 import java.util.UUID;
 
 /**
- * Converts the player to the Profile
+ * Default implementation for {@link ProfileProvider}.
  */
-@SuppressWarnings({"PMD.ClassNamingConventions", "PMD.CommentRequired"})
-public final class PlayerConverter {
-
-    private PlayerConverter() {
-    }
-
+public class UUIDProfileProvider implements ProfileProvider {
```

Comment:
It seems like you swapped Profile and OfflineProfile implementations, I would prefere you swap them back to reduce diff.

Reply:
Swapped them again